modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/client/DefaultExtensionsClient.java

```diff
@@ -152,6 +169,38 @@ private OperationMessageProcessor createProcessor(String extensionName, String o
     }
   }
 
+  private UninitializedOperationMessageProcessor createFrulaOperationmessageprocessor(String extensionName,
+                                                                                      String operationName,
+                                                                                      OperationParameters parameters,
+                                                                                      OperationMessageProcessor cachedOperationMessageProcessor) {
+
+    ExtensionModel extension = findExtension(extensionName);
+    OperationModel operation = findOperation(extension, operationName);
+    ConfigurationProvider configurationProvider = parameters.getConfigName().map(this::findConfiguration).orElse(null);
+    Map<String, ValueResolver> resolvedParams = resolveParameters(parameters.get(), getEvent());
+
+    try {
+      UninitializedOperationMessageProcessor uninitializedOperationMessageProcessor =
+          new UninitializedOperationMessageProcessor(extension, operation, policyManager, registry,
+                                                     cachedOperationMessageProcessor,
+                                                     resolvedParams, muleContext,
+                                                     getConfigurationProvider(resolvedParams, configurationProvider));
+
+      //initialiseIfNeeded(uninitializedOperationMessageProcessor, muleContext);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7719&issues=AWk7fTJKotIXnmsUoDr_&open=AWk7fTJKotIXnmsUoDr_)

[ISSUE_KEY]: AWk7fTJKotIXnmsUoDr_